src/modules/promocode/promo.service.spec.ts

```diff
@@ -0,0 +1,120 @@
+import type { TestingModule } from '@nestjs/testing';
+import { Test } from '@nestjs/testing';
+
+import {
+    listOfPromocodes,
+    pageOptions,
+    promocode,
+    promocodeName,
+    promocodeToAdd,
+    promocodeToRemove,
+} from './promo.fixture';
+import { MockPromoCodeService } from './promo.mock.service';
+import { PromocodeService } from './promo.service';
+
+class ClientGrpcMock {
```

Comment:
nit: can be also moved to mock file

Reply:
@dianager  I have a limit on the number of classes in a file. The max-classes-per-file setting is currently set to 1. Should I change it or move ClientGrpcMock to a different file, or left as is?

Reply:
I would better move to promo.client.mock.ts file, in case in future you will need it in other tests